shared/chat/conversation/input.desktop.js

```diff
@@ -5,12 +5,15 @@ import {globalColors, globalMargins, globalStyles} from '../../styles'
 import {Picker} from 'emoji-mart'
 import {backgroundImageFn} from '../../common-adapters/emoji'
 
-import type {Props} from './'
+import type {Props} from './input'
 
 type State = {
   emojiPickerOpen: boolean,
+  text: string,
 }
 
+const _cachedInput: {[key: ?string]: ?string} = { }
```

Comment:
I guess this is okay, but this is the first time we do something like this I think. and it's better to not have state strewn around.